.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 orbs:
   node: circleci/node@5.0
   php: circleci/php@1.1
-  wp-svn: studiopress/wp-svn@0.1
+  wp-svn: studiopress/wp-svn@dev:alpha
```

Comment:
TODO: revert when https://github.com/studiopress/wp-svn-orb/pull/17 is merged and I tag a new release